### PR TITLE
feat: move send to wallet logic to new bookmark component

### PR DESCRIPTION
**Pull request by 0xsebn**
Task: 
This task solves:
https://lifi.atlassian.net/browse/LF-2839

and

https://lifi.atlassian.net/browse/LF-5322

Task was initially linked to what is now the epic.
https://lifi.atlassian.net/jira/software/c/projects/LF/boards/15?modal=detail&selectedIssue=LF-4928&assignee=6398600d3c9bcd363977188e

Created bookmark page where user is able to input wallet address and on clicking confirm gets sent to main page.

This is part 1 of creating the bookmark functionality. The component to switch from "Custom" to "Bookmarks" is hidden atm. 

image

### CODE REVIEW

**Comment by 0xsebn** [comment]
For the first milestone where the goal is to display exchange warning message missing the 

-styling of the SendToWallet component (switching it to a clickable button instead of an input)
-Hiding the bookmarks button on the BookmarkLanding component
-removing unnecessary code from SendToWallet since its logic has been moved to BookmarkLanding

**Comment by dennyscode** [comment]
Into how many pieces will this ticket be split? In the ticket itself, it´s not mentioned that we will do it in multiple iterations. 
But from your comments, it seems to be on point.

**Review by chybisov — CHANGES_REQUESTED** [review]

File: packages/widget/src/components/SendToWalletButton/ShowSendToWalletButton.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { FormKey, useWallet, useWidgetConfig } from '../../providers';
 import { useSendToWalletStore, useSettings } from '../../stores';
 import { DisabledUI, HiddenUI, RequiredUI, WidgetEvent } from '../../types';
 
-export const SendToWalletButton: React.FC = () => {
+export const ShowSendToWalletButton: React.FC = () => {
```
Comment (chybisov) [inline packages/widget/src/components/SendToWalletButton/ShowSendToWalletButton.tsx:10]: ShowSendToWalletButton -> SendToWalletToggleButton

**Review by chybisov — APPROVED** [review]

**Comment by 0xsebn** [comment]
For any other person possibly reviewing this atm, hopped on a call with Eugene and he gave me some more comments. Will put the wait tag for now till I finish those

**Comment by chybisov** [comment]
Finished in #168

**Pull request closed without merging**